force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.html

```diff
@@ -0,0 +1,12 @@
+<template>
+    <div class="slds-grid slds-wrap">
+        <nav class="slds-nav-vertical slds-nav-vertical_compact">
+            <template for:each={edaNavigationViewModel.navigationSections} for:item="navigationSection">
+                <c-eda-settings-navigation-section
```

Comment:
I ended up having to break this out in order to handle the active state of a page from a data-driven perspective.